### ISSUE TITLE
Testnet config file missing and causing docker container to fail

### ISSUE THREAD

**Issue by melmccann** [cardano-foundation/cardano-wallet#1832]
I did a full system prune and cleared all volumes. pulled cardano-wallet and ran "NETWORK=testnet docker-compose up"
I am getting the following error in the wallet logs.

Usage: cardano-wallet-byron COMMAND
  Cardano Wallet Command-Line Interface (CLI)
Invalid argument `/config/pdnsmv1gi6llr92fk7sgs3j2gcphm6fm-shelley-testnet-genesis.json' 

Container will not start in testnet mode.

**Comment by piotr-iohk** [comment]
It seems that currently within the cardano-node docker image there are bunch of genesis files:
```
7106qjwral616j6p5049m7hhi5ih0r33-testnet-genesis.json
b25q8nqwqlhh72q54ygir5mj0rkgjbvs-alpha1-genesis.json
bdjsaqfxfyq0f5pdvbzxwa4drac9g5mz-mainnet-ci-genesis.json
ciwax7jcb8g22kr90vy8vax4bfx0r450-mainnet-genesis-dryrun-with-stakeholders.json
dkbnz9irv7s2bmff3vj8yhsplpynhzdw-shelley-staging-genesis.json
fjim537saaiigwmbrdcdcpi00gy7kxg3-latency-tests-genesis.json
fz47fp0ssycfgs9r7345q9rbvclid5nd-selfnode-genesis.json
h6h587v0sdas7m2yb81r8ixn6zdv6wyh-shelley-staging-short-genesis.json
pdnsmv1gi6llr92fk7sgs3j2gcphm6fm-shelley-testnet-genesis.json
qd60rqsk0rjdrckg2js8dgl9jh21bd1j-mainnet-genesis.json
rpw8qsf3qhk5cw4z4pjwvggkf650sdq5-genesis.json
v8lb35176p73f6ksp7p5rhkqmygi1zj9-ff-genesis.json
yn4hv5h5kmmhcqf4amysbp85r8sn6mgw-alpha2-genesis.json
zpb044kq24h6qkywl9vxg8d03sb45za2-shelley-qa-genesis.json
```
when doing NETWORK=testnet the wallet takes `pdnsmv1gi6llr92fk7sgs3j2gcphm6fm-shelley-testnet-genesis.json`

and should be taking `7106qjwral616j6p5049m7hhi5ih0r33-testnet-genesis.json` since it is the one correct for Byron testnet...

**Comment by KtorZ** [comment]
Changing line 25 from:

```
        ([[ $$NETWORK == \"testnet\" ]] && $$CMD --testnet /config/*testnet-genesis.json)
```

to 

```
        ([[ $$NETWORK == \"testnet\" ]] && $$CMD --testnet /config/7106qjwral616j6p5049m7hhi5ih0r33-testnet-genesis.json)
```

should fix it, the genesis shouldn't really change hash anyway. 
Yet locally, I am running into some other issues when trying out `cardano-node:latest`

```
ERROR: for 3a22238d73f9_3a22238d73f9_cardanowallet_cardano-node_1  Cannot start service cardano-node: OCI runtime create failed: container_linux.go:349: starting container process caused "exec: \"/nix/store/f95y53rymd67kb373arh57npbkxdwjyy-entry-point/bin/entry-point\": stat /nix/store/f95y53rymd67kb373arh57npbkxdwjyy-entry-point/bin/entry-point: no such file or directory": unknown
```

**Comment by piotr-iohk** [comment]
Fixed by #1934.
